### PR TITLE
Tests the response of unprocessable data for the create action

### PR DESCRIPTION
**Pull request by J-pilon**

### CODE REVIEW

File: spec/requests/articles_spec.rb [hunk 1]
```diff
@@ -41,10 +41,26 @@
...
 
   describe "POST /articles" do
-    it "adds the new article" do
+    context "successfully redirects and" do
+      it "adds the new article" do
 
-      post "/articles", :params => {:article => {:title => "Article Title", :body => "This is the article body"}}
-      expect(response).to have_http_status(302)
+        post "/articles", :params => {:article => {:title => "Article Title", :body => "This is the article body"}}
+        expect(response).to have_http_status(302)
```
Comment (bchangg) [inline spec/requests/articles_spec.rb:48]: Ooh where does this `response` come from? Unfamiliar with this!

Comment (bchangg) [reply spec/requests/articles_spec.rb:48]: Oh interesting, does it just come with the `type: :request` test?

Comment (J-pilon) [reply spec/requests/articles_spec.rb:48]: "response is a variable representing the HTTP response we receive after making a HTTP request, its class is [ActionDispatch::TestResponse](https://api.rubyonrails.org/classes/ActionDispatch/TestResponse.html), we can use its’ body method to get the response body to check its content, status attribute to get the HTTP status code, and headers attribute to get the HTTP headers." - https://rubyyagi.com/rspec-request-spec/#create-a-health-check-controller-to-return-fixed-json-response

This article helped me understand.

**Comment by misterhtmlcss** [comment]
@J-pilon what edge cases do you think you could have missed?

**Pull request merged**